Easy/02 - First Factorial.js

```diff
@@ -13,13 +13,17 @@ function FirstFactorial (num) {
   return num;
 }
 
-// Recursive Solution
-function FirstFactorial (num) {
-  if (num === 1) {
-    return num;
+// Solution using recursive function
+function FirstFactorial(num) { 
+    if(num === 0) {
+       return 1;
+    }
+    else if(num < 0 ) {
```

Comment:
As the "step by step" solution doesn't return undefined for cases less than 0, the recursive solution shouldn't either. I also prefer independent `if` clauses rather than `else if`, though if you do use `else if` please have the `else if` on the same line as the closing brace of the previous `if`.

All `if`/`else if` statements should have a space before their case, as well.